### PR TITLE
Add docs about theme_barceloneta subtemplate

### PR DESCRIPTION
**Pull request by erral**

### CODE REVIEW

File: docs/theme_barceloneta.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+====================================================================
+Creating a Plone theme package based on Barceloneta inside a package
+====================================================================
+
+With this ``template``, you can add a Barceloneta based template to a Plone package. This will add a copy of Barceloneta theme including a full Grunt setup and a copy of all Barceloneta resources.
```
Comment (svx) [inline docs/theme_barceloneta.rst]: Please follow our style-guide: https://docs.plone.org/about/contributing/documentation_styleguide.html

One sentence per line

File: docs/theme_barceloneta.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+If you like it a bit more modular or you don't want add the Barceloneta Theme as a start, you might want to use the :doc:`theme subtemplate </theme>` instead!
+
+.. code-block:: sh
```
Comment (svx) [inline docs/theme_barceloneta.rst]: Please follow our `rst` style-guide: https://docs.plone.org/about/contributing/rst-styleguide.html

We use `shell`

File: docs/theme_barceloneta.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
+====================================================================
+Creating a Plone theme package based on Barceloneta inside a package
+====================================================================
+
+With this ``template``, you can add a Barceloneta based template to a Plone package. This will add a copy of Barceloneta theme including a full Grunt setup and a copy of all Barceloneta resources.
+
+If you like it a bit more modular or you don't want add the Barceloneta Theme as a start, you might want to use the :doc:`theme subtemplate </theme>` instead!
```
Comment (svx) [inline docs/theme_barceloneta.rst]: " A bit more " is wordry :)

File: docs/theme_barceloneta.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+.. code-block:: sh
+
+    mrbob -O plonetheme.blacksea bobtemplates.plone:addon
+
+then change into the created folder ``plonetheme.blacksea`` and add a new theme:
+
+.. code-block:: sh
```
Comment (svx) [inline docs/theme_barceloneta.rst]: Same here

File: docs/theme_barceloneta.rst [hunk 5]
```diff
@@ -0,0 +1,30 @@
+====================================================================
+Creating a Plone theme package based on Barceloneta inside a package
```
Comment (svx) [inline docs/theme_barceloneta.rst]: Please follow our style-guide about headings: https://docs.plone.org/about/contributing/documentation_styleguide.html

Comment (erral) [reply docs/theme_barceloneta.rst]: I don't understand this comment, what should I do here? I copied the header from other doc file like: 

https://raw.githubusercontent.com/plone/bobtemplates.plone/master/docs/theme.rst

File: docs/theme_barceloneta.rst (hunk 4 above)
Comment (erral) [reply docs/theme_barceloneta.rst]: Then, I will change this all over the documentation

File: docs/theme_barceloneta.rst (hunk 2 above)
Comment (erral) [reply docs/theme_barceloneta.rst]: Then, I will change this all over the documentation

File: docs/theme_barceloneta.rst (hunk 5 above)
Comment (svx) [reply docs/theme_barceloneta.rst]: We have a section about headings in our style-guide, unfortunately this tents to be forgotten.

So at least for new files, I try to remind people about the guidelines and we adjusting slowly already written documentation.

If you for example browse the docs.plone.org you will see that already through the whole documentation.

From our style-guide:

| Headings And Filenames
| Start capitals – capitalization of all words, regardless of the part of speech.

**Comment by erral** [comment]
I will close this PR and add a new one then.

**Comment by erral** [comment]
I see that @MrTango already added the piece of doc I was going to add there, so it's OK on my side, thanks

**Comment by MrTango** [comment]
@erral I just took over some of the existing docs, please have a look if it's complete and feel free to add whats missing.

**Comment by erral** [comment]
I have read them and they're OK. it's indeed  what I was going to write

**Pull request closed without merging**